bendigobank.rb

```diff
@@ -0,0 +1,87 @@
+require 'watir'
+require 'json'
+require 'nokogiri'
+require 'date'
+require_relative 'account'
+require_relative 'transaction'
+
+class Bendigobank
+
+  def self.execute
+    connect
+    fetch_accounts(connect)
+    fetch_transactions(connect)
+  end
+
+  def self.connect
+    Watir.logger.ignore :deprecations
+    browser = Watir::Browser.new :firefox
+    browser.goto 'http://demo.bendigobank.com.au'
+    browser.button(name: 'customer_type').click
+    return browser
+  end
+
+  def self.fetch_accounts(browser)
+    html = Nokogiri::HTML.fragment(browser.div(class: "_23LTBXgogQ").html)
```

Comment:
Use ccs class here value `_23LTBXgogQ` will be changed at the next bank session.
```
    html = Nokogiri::HTML.fragment(browser.div(class: "accounts-list").html)
    parse_accounts(html)
```